core/src/main/java/org/jclouds/proxy/ProxyForURI.java

```diff
@@ -84,9 +81,20 @@ public PasswordAuthentication getPasswordAuthentication() {
             Authenticator.setDefault(authenticator);
          }
          return proxy;
-      } else {
-         return Proxy.NO_PROXY;
       }
+      if (config.canUseProxyFromJvm()) {
+         return getDefaultProxy(endpoint);
+      }
+      if (config.useSystem()) {
+         System.setProperty("java.net.useSystemProxies", "true");
```

Comment:
> there is a comment on the property in Constants to that effect

I was thinking more along the lines of an implementer hint? Something like `// probably does nothing - see comment in Constants...`?